extras/confd/templates/pgbouncer.tmpl

```diff
@@ -0,0 +1,17 @@
+[databases]
+{{range gets "/members/*"}}{{$data := json .Value}}{{if eq $data.role "master" }}{{ $hostport := base (replace (index (split $data.conn_url "/") 2) "@" "/" -1)}}{{ $host := base (index (split $hostport ":") 0)}}{{ $port := base (index (split $hostport ":") 1)}}employees = host={{ $host }} port={{ $port }} dbname=employees pool_size=10{{end}}{{end}}
```

Comment:
We should not rely on the role written in the member json.
In order to build the configuration we should do the following:
1. get /leader
2. get /member/$leader

Also I don't like hard-coded database name, "employees", instead we could use pgbouncer [autodb](https://pgbouncer.github.io/config.html#section-databases) feature.
Basically we should specify something like: `* = host={{ $host }} port={{  $port }}`

Reply:
Ok, I'll correct in this way. The database name is a mistake, indeed, the sample tmpl should use autodb